exampleCourse/questions/element/multipleChoice/question.html

```diff
@@ -196,3 +196,31 @@
 
     </div>
 </div>
+
+<div class="card my-2">
+    <div class="card-header">
+        Part 7
+    </div>
+
+    <div class="card-body">
+
+        <pl-question-panel>
+            <p>Here, you can observe the behavior of the <code>score</code> attribute in <code>pl-answer</code>.
+            The score for a question the percentage of points given to it if selected (default is 0 for
+            incorrect answers and 1 for correct ones.)</p>
+
+            <p> What is the color of the sky? </p>
+        </pl-question-panel>
+
+        <pl-multiple-choice answers-name="sky-color8">
+            <pl-answer correct="true" feedback="Correct! The sky is indeed blue.">Blue</pl-answer>
+            <pl-answer>Pink</pl-answer>
+            <pl-answer score=0.5>Purple</pl-answer>
+            <pl-answer feedback="Not quite" score=0.1>Orange</pl-answer>
+            <pl-answer>Yellow</pl-answer>
+            <pl-answer score=0.3>Brown</pl-answer>
+            <pl-answer score=0.2>Red</pl-answer>
+          </pl-multiple-choice>
```

Comment:
Nit, consistent indentation:
```suggestion
        </pl-multiple-choice>
```